src/core.ts

```diff
@@ -179,20 +179,24 @@ export function createGetIronSession(
 
     Object.defineProperties(session, {
       save: {
-        value: async function save() {
+        value: async function save(saveOptions: IronSessionOptions) {
           if ("headersSent" in res && res.headersSent === true) {
             throw new Error(
               `iron-session: Cannot set session cookie: session.save() was called after headers were sent. Make sure to call it before any res.send() or res.end()`,
             );
           }
+
+          // Use options as base and extend with function argument
+          saveOptions = Object.assign({}, saveOptions);
```

Comment:
I think you are missing the `options` here. 
```
          saveOptions = Object.assign({}, options, saveOptions);
```

Reply:
Fixed! Also made the save/destroy options optional.